app/assets/javascripts/reducers/stories.js

```diff
@@ -12,7 +15,47 @@ const storiesReducer = (state = initialState, action) => {
   switch (action.type) {
     case actionTypes.RECEIVE_STORIES:
       return action.data;
+    case actionTypes.CREATE_STORY:
+      if (isCreating(state)) {
+        const story = state.find(isNew);
+
+        if (!creatingInAnotherColumn(story, action.attributes.state)) {
```

Comment:
Maybe we could make it a little more explicit why this is necessary. What do you think about renaming the method?